<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hs_Node_setting</class>
 <widget class="QWidget" name="Hs_Node_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="NodeName">
       <property name="text">
        <string>节点名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nodename_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="UpLimit">
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="uplimit_doubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="DownLimit">
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="downlimit_doubleSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
